citadel/Manipulating_models.md

```diff
@@ -0,0 +1,148 @@
+
+![World frame alignment - rotation](img/rotate_worldframe.png)
+
+### Enter Custom Snap Values
+
+When translating or rotating, you can "snap" an entity's movement to preset increments by holding `Ctrl` and then clicking and dragging.
+By default, the snap value is 1 meter for translation and 45° for rotation.
+
+Translational snaps will snap relative to the world frame, whereas rotational snaps will snap relative to the object's current orientation.
+
+![Snap rotation](img/snap.gif)
+
+You can customize snap values by clicking on the furthest-right magnet icon in the Transform Control plugin.
+
+![Custom snap values](img/custom_snap.png)
+
+Try holding `Shift` and `Ctrl` simultaneously to snap a model along the world frame is if isn't already aligned.
+
+## Component Inspector
+
+The Component Inspector plugin is included in `shapes.sdf`.
+With it, you can select an entity to display some of its attributes, including its pose, whether or not wind is acting on it, whether or not it's static, and more.
+The plugin also displays the gravitational and magnetic fields of your world.
+
+Expanding the `Pose` attribute will show the coordinate values for a selected object.
+The values will update as the object is moved around (translated or rotated) while the simulation is playing.
+
+![Updating pose values](img/pose_value.gif)
+
+You can also pause the plugin so an object's pose values don't update in the inspector while the simulation is playing.
+
+Additionally, the Component Inspector can be locked on one object so you can open another Component Inspector to simultaneously observe another object.
+
+![Two Component Inspectors](img/pose2.gif)
+
+## View Angle
+
+The View Angle plugin does not come pre-loaded with `shapes.sdf`, so you will have to select it from the plugins button on the toolbar.
+
+You can choose which angle you want to view the scene from, relative to a selected entity or the world frame if no entity is selected.
+The home button will return you to the original view pose from when the scene was loaded.
+
+You can also select multiple entities to face simultaneously from each view angle.
+
+![View angle for multiple entities](img/view.gif)
+
+## Align Tool
+
+Like the View Angle plugin, the Align Tool is also not pre-loaded and will also have to be selected from the plugins button on the toolbar.
+
+The Align Tool aligns a model up with the center/origin of another model, or with something besides a model, like a link for example.
+It can't, however, align a link or anything other than the top-level model up with something else.
+
+With `Relative to First` selected, select two models using `Ctrl` + click to see the second model align to the first along either it's x, y or z axis.
+Hovering over the buttons for each axis will show a preview of where the model would end up if aligned along that axis.
+
+You can align more that two models, just select the others with `Ctrl` + click.
```

Comment:
that -> than, same for blueprint tutorial

Reply:
There isn't an align tool for blueprint right?

Reply:
> There isn't an align tool for blueprint right?

Correct